clouddriver-azure/src/test/groovy/com/netflix/spinnaker/clouddriver/azure/resources/servergroups/deploy/AzureServerGroupResourceTemplateSpec.groovy

```diff
@@ -2814,7 +2814,7 @@ class AzureServerGroupResourceTemplateSpec extends Specification {
               "autoUpgradeMinorVersion" : true,
               "settings" : {
                 "protocol" : "https",
-                "port" : "7000",
+                "port" : 7000,
```

Comment:
This looks like a breaking change (string to int)

Reply:
Sorry, I misinterpreted this. This is the request that will be shipped to Azure, which must be an int.